oh.ps1

```diff
@@ -98,14 +98,15 @@ $global:ProgressPreference= 'SilentlyContinue'
 ##################### OH general configuration ####################
 
 # -> OH_PATH is the directory where Open Hospital files are located
-# OH_PATH="c:\Users\OH\OpenHospital\oh-1.13"
+# OH_PATH="c:\Users\OH\OpenHospital\oh"
 
 # set OH mode to PORTABLE | CLIENT | SERVER - default set to PORTABLE
 #$script:OH_MODE="PORTABLE"
 
 # language setting - default set to en
-$script:OH_LANGUAGE_LIST="en|fr|es|it|pt|ar"
-$script:OH_LANGUAGE="en" # default
+$script:OH_LANGUAGE_LIST= @("ar","de","en","es","fr","it","pt","sq")
+$script:OH_LANGUAGE_LIST_INFO=("arabic","german","english","spanish","french","italian","portuguese","albanian")
```

Comment:
same comment as there is for line 53 in oh.sh below

Reply:
above ? :-) :-)

Reply:
See https://github.com/informatici/openhospital-gui/pull/1959